fontbe/src/glyphs.rs

```diff
@@ -258,14 +265,49 @@ impl Work<Context, Error> for GlyphWork {
                 };
                 context.set_glyph(name.clone(), base_glyph.clone().into());
 
-                (name, point_seqs_for_simple_glyph(ir_glyph, instances))
+                let mut contour_end = 0;
+                let mut contour_ends = Vec::with_capacity(base_glyph.contours().len());
+                for contour in base_glyph.contours() {
+                    contour_end += contour.len();
+                    contour_ends.push(contour_end);
+                }
+                (
+                    name,
+                    point_seqs_for_simple_glyph(ir_glyph, instances),
+                    contour_ends,
+                )
             }
         };
 
+        let coords = point_seqs.get(default_location).ok_or_else(|| {
+            Error::GlyphError(ir_glyph.name.clone(), GlyphProblem::MissingDefault)
+        })?;
+
+        // TODO real values
+        let tolerance = 0.1;
```

Comment:
I think fonttools varLib uses tolerance=0.5, and it doesn't look like it allows to override that from the public varLib.build(), so we should match that I think:

https://github.com/fonttools/fonttools/blob/65bc6105f7aec3478427525d23ddf2e3c8c4b21e/Lib/fontTools/varLib/__init__.py#L239

Reply:
good catch, making it so